bitbadges/chain.json

```diff
@@ -0,0 +1,100 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "bitbadges",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://bitbadges.io/",
+  "pretty_name": "BitBadges",
+  "chain_id": "bitbadges-1",
+  "bech32_prefix": "cosmos",
+  "daemon_name": "bitbadgeschaind",
+  "node_home": "$HOME/.bitbadgeschaind",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "ubadge",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0.00025,
+        "average_gas_price": 0.0025,
+        "high_gas_price": 0.025
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "ustake"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/bitbadges/bitbadgeschain/",
+    "recommended_version": "v1.0-mainnet",
+    "compatible_versions": [
+      "v1.0-mainnet"
+    ],
+    "versions": ["v1.0-mainnet"],
```

Comment:
The versions array contains objects, not strings. Please redefine v1.0-mainnet as a version object